pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining.detailed-output

```diff
@@ -152,6 +146,15 @@ Message: Node build0-gstfj-ci-tests-worker-c-dcz9p is degraded
     nodes: build0-gstfj-ci-tests-worker-c-dcz9p
   Description: failed to drain node: build0-gstfj-ci-tests-worker-c-dcz9p after 1 hour. Please see machine-config-controller logs for more information
 
+Message: Cluster Version version is failing to reconcile (ClusterOperatorDegraded)
+  Since:       0s
```

Comment:
Also, do we want to throttle when this shows up to the user? For CO conditions we wait for the condition to persist for some time before we start showing insights for them.

Reply:
We throttle ClusterOperator conditions because we don't maintain those components, and sometimes it takes us a few years to talk their maintainers into denoising.  We maintain the cluster-version operator, so if `Failing` is noisy, we can denoise server-side as quickly as we like.

Reply:
OK, potentially throttling some `Failing` causes is better to do in CVO, I agree. Good point.